tools/lsp/main.rs

```diff
@@ -102,6 +105,22 @@ struct Format {
     inline: bool,
 }
 
+#[cfg(feature = "preview-engine")]
+#[derive(Args, Clone, Debug)]
+struct LivePreview {
+    /// Run remote controlled by the LSP
+    #[arg(long)]
+    remote_controlled: bool,
```

Comment:
there is an attribute `hide` or `hidden` that can be set so it is not part of the help as this is an internal thing.

I'm actually wondering if the all live-preview command shouldn't be all internal.

Not sure if we want some other command to open the UI with a given component like what is done in #7576 then it would need to be public. But this is not ready for that IMHO. So in the mean time the whole thing should be internal.